.circleci/create_circleci_config.py

```diff
@@ -517,7 +516,7 @@ def job_name(self):
         "pip install --upgrade --upgrade-strategy eager pytest pytest-sugar",
         "pip install -U --upgrade-strategy eager natten",
         "find -name __pycache__ -delete",
-        "find . -name \*.pyc -delete",
+        "find . -name \\*.pyc -delete",
```

Comment:
I would personally check if this new (format) command would work well on a machine's bash commandline.